test/e2e/service.go

```diff
@@ -230,6 +230,130 @@ var _ = Describe("Services", func() {
 		validateEndpointsOrFail(c, ns, serviceName, map[string][]int{})
 	})
 
+	It("should be able to up and down services", func() {
+		ns := namespaces[0]
+		numPods, servicePort := 3, 80
+
+		podNames1, svc1IP, err := start_serve_hostname_service(c, ns, "service1", servicePort, numPods)
```

Comment:
Actually, I'm tearing them down:
- service1 is teared down in line 253
- service2 and service3 are teared down at the very end
 I'm not tearing them down in "defer" way (as above), because I'm explicitly stopping service1 as part of the test.

 Does it sound ok?